Updating Product OS sidebar (#11067)
PostHog/posthog.com: contents/docs/getting-started/next-steps.mdx

---
title: Next steps
sidebar: Docs
showTitle: true
---

## 1. Migrate historical data from another tool

Are you migrating from another tool like PostHog?

1. Congratulations on a great descision. Go you.
2. Head to our [docs on historial migrations](/docs/migrate).

> ## 2. Add PostHog to your website

Author: Disagree. I think `install.mdx` already has enough detail expands and FYIs. I think it's still true that most people install PostHog on their product first. This is the right time to remind people why they should do this + the benefits of doing so.

It's cheap, too. You can send us [anonymous events](/docs/data/anonymous-vs-identified-events) from your website, which are up to 4x cheaper than the identified events used for product analytics.

You can then use our [web analytics](/docs/web-analytics) dashboard to track your top pages, how people find your website, and your [web vitals](/docs/web-analytics/web-vitals).

Read our [web analytics vs product analytics](/docs/web-analytics/web-vs-product-analytics) explainer for more.

## 3. Invite your team

PostHog is way more powerful when you're using it with your teammates.

To invite them, go to your instance homepage, click the **Invite** button, and add their email. 

This will add them to your organization with access to all your projects.

//TKTK update screenshot

![Invite](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/next-steps-after-installing/invite.png)

You can change their access levels in your organization settings, and set some projects to private if required.

You'll likely want to [filter out events from internal users](/tutorials/filter-internal-users) if you do this.

> 

Reviewer: We can tie in some of Danilo's getting started content here once ready.

Head to the [data pipelines](/docs/cdp) docs for more on how you can use PostHog as your customer data platform.

You can also configure [batch exports](/docs/cdp/batch-exports/s3) to a data warehouse.

## 6. Import revenue and customer data into our data warehouse

When you choose PostHog, you also get a built-in data warehouse.

You can then [link your most valuable data](/docs/data-warehouse/join) from sources like your CRM, payment processor, or database, combine it with the event data already in PostHog, and query across all of it.

We currently offer fully-managed import integrations for the likes of [Stripe](/docs/cdp/sources/stripe), [Hubspot](/docs/cdp/sources/hubspot), and [Salesforce](/docs/cdp/sources/salesforce).

We also support important data from most popular data warehouses and databases, such as [Postgres](/docs/cdp/sources/postgres), [BigQuery](/docs/cdp/sources/bigquery), and [Amazon S3](/docs/cdp/sources/s3).

See the full [list of PostHog data sources](/docs/cdp/sources) for more. 

> ## 7. Figure out what to track in PostHog

Reviewer: I don't think this is the right framing, maybe we should say "Learn how to win with PostHog" or "Use PostHog to help build a successful product." What to track feels less important than using what you are tracking already to make you successful.

> ## 7. Figure out what to track in PostHog

Author: ```suggestion
## 7. Learn how to win with PostHog
```

Reviewer 2: I like that

> PostHog has [autocapture](/docs/product-analytics/autocapture), so you're already collecting useful data from the moment you install our snippet.
> 
> But you'll still need to setup some custom events to get the most value from PostHog.

Author: ```suggestion
PostHog exists to help you build more successful products and we provide a lot of tools to get the job done.
```

If you have no idea where to start, we recommend reading our guide to [Winning with PostHog](/docs/new-to-posthog/getting-hogpilled). It will guide you through:

- [Creating a tracking plan](/docs/new-to-posthog/getting-hogpilled) by defining your North Star metric.
- [How to track and measure activation](/docs/new-to-posthog/activation) – aka one of the most important metrics ever.
- [How to think about and track retention](/docs/new-to-posthog/retention) of users and customers.
- [Capturing revenue in PostHog](/docs/new-to-posthog/revenue), so you can monitor the health of your whole business.

![a complete dashboard](https://res.cloudinary.com/dmukukwp6/image/upload/hogflix_c681b6891d.png)

> ## 8. Start testing in production

Reviewer: Could we make this a point about both feature flags and experiments? The title has a sort of double meaning. Maybe this is cramping too many points in one section.

---
title: Next steps
sidebar: Docs
showTitle: true
---

## 1. Migrate historical data from another tool

Are you migrating from another tool like PostHog?

1. Congratulations on a great descision. Go you.
2. Head to our [docs on historial migrations](/docs/migrate).

## 2. Add PostHog to your website

Installing PostHog on both your app and website enables you track the entire funnel of a user through acquisition, activation, retention, and revenue.

It's cheap, too. You can send us [anonymous events](/docs/data/anonymous-vs-identified-events) from your website, which are up to 4x cheaper than the identified events used for product analytics.

You can then use our [web analytics](/docs/web-analytics) dashboard to track your top pages, how people find your website, and your [web vitals](/docs/web-analytics/web-vitals).

Read our [web analytics vs product analytics](/docs/web-analytics/web-vs-product-analytics) explainer for more.

## 3. Watch your first session replay

> Watching people using your product is like magic.

Reviewer: ```suggestion
Watching people using your product is magic.
```

Once you've [enabled session replay](/docs/session-replay/installation) in your project, head to the [what to watch tab](https://us.posthog.com/replay/templates) in session replay to watch how users naviagte specific parts of your app.

Warning: watching replays is addictive, but incredibly useful.

## 4. Import revenue and customer data into our data warehouse

When you choose PostHog, you also get a built-in data warehouse.

> You can then [link your most valuable data](/docs/data-warehouse/join) from sources like your CRM, payment processor, or database, combine it with the event data already in PostHog, and query across all of it.

Reviewer: ```suggestion
You can use it to [link your most valuable data](/docs/data-warehouse/join) from sources like your CRM, payment processor, or database, combine it with the event data already in PostHog, and query across all of it.
```
